schema-lending-extended.graphql

```diff
@@ -0,0 +1,1173 @@
+  cumulativeSupplySideRevenueUSD: BigDecimal!
+
+  " Hourly revenue generated by the market, accrued to the supply side. "
+  hourlySupplySideRevenueUSD: BigDecimal!
+
+  " All revenue generated by the market, accrued to the protocol. "
+  cumulativeProtocolSideRevenueUSD: BigDecimal!
+
+  " Hourly revenue generated by the market, accrued to the protocol. "
+  hourlyProtocolSideRevenueUSD: BigDecimal!
+
+  " All revenue generated by the market. "
+  cumulativeTotalRevenueUSD: BigDecimal!
+
+  " Hourly revenue generated by the market. "
+  hourlyTotalRevenueUSD: BigDecimal!
+
+  " Current balance of all deposited assets (not historical cumulative), in USD. Same as pool TVL. "
+  totalDepositBalanceUSD: BigDecimal!
+
+  " Sum of all deposits made in a given hour, in USD "
+  hourlyDepositUSD: BigDecimal!
+
+  " Sum of all historical deposits in USD (only considers deposits and not withdrawals) "
+  cumulativeDepositUSD: BigDecimal!
+
+  " Current balance of all borrowed/minted assets (not historical cumulative), in USD. "
+  totalBorrowBalanceUSD: BigDecimal!
+
+  " Sum of all borrows/mints made in a given hour, in USD "
+  hourlyBorrowUSD: BigDecimal!
+
+  " Sum of all historical borrows/mints in USD (i.e. total loan origination) "
+  cumulativeBorrowUSD: BigDecimal!
+
+  " Total assets liquidated in a given hour, in USD. "
+  hourlyLiquidateUSD: BigDecimal!
+
+  " Sum of all historical liquidations in USD "
+  cumulativeLiquidateUSD: BigDecimal!
+
+  " Total assets withdrawn on a given hour, in USD. "
+  hourlyWithdrawUSD: BigDecimal!
+
+  " Total assets repaid on a given hour, in USD. "
+  hourlyRepayUSD: BigDecimal!
+
+  ##### Token Balances #####
+
+  " Amount of input token in the market. "
+  inputTokenBalance: BigInt!
+
+  " Price per share of input token in USD "
+  inputTokenPriceUSD: BigDecimal!
+
+  " Total supply of output token "
+  outputTokenSupply: BigInt!
+
+  " Price per share of output token in USD "
+  outputTokenPriceUSD: BigDecimal!
+
+  " Amount of input token per full share of output token. Only applies when the output token exists "
+  exchangeRate: BigDecimal
+
+  " Per-block reward token emission as of the current block normalized to a day (not hour), in token's native amount. This should be ideally calculated as the theoretical rate instead of the realized amount. "
+  rewardTokenEmissionsAmount: [BigInt!]
+
+  " Per-block reward token emission as of the current block normalized to a day (not hour), in USD value. This should be ideally calculated as the theoretical rate instead of the realized amount. "
+  rewardTokenEmissionsUSD: [BigDecimal!]
+}
+
+##############################
+##### Account-Level Data #####
+##############################
+
+type Account @entity {
+  " { Account address } "
+  id: ID!
+  
+  " Number of open positions this account has "
+  positionCount: Int!
+
+  " All open positions that belong to this account "
+  positions: [Position!]!
```

Comment:
Just sanity checking that old & closed positions will be in here. If I were to ping all positons/events for an Account, I'd have a full view of their history?